src/renderer/components/dialog/dialog.tsx

```diff
@@ -167,7 +167,7 @@ export class Dialog extends React.PureComponent<DialogProps, DialogState> {
 
     if (animated) {
       dialog = (
-        <Animate enter={this.isOpen} name="opacity-scale">
+        <Animate enter={this.isOpen} name="opacity-scale" enterDuration={200} leaveDuration={200}>
```

Comment:
Why `200` for dialogs? 
I'd better to use predefined values inside `*.css` for all animations and apply to element via `className`.

Reply:
Sure, let's use our basic predefined values.